packages/portal/src/plugins/europeana/set.js

```diff
@@ -125,6 +129,10 @@ export default class EuropeanaSetApi extends EuropeanaApi {
    * @return {Object} API response data
    */
   update(id, data, params = {}) {
+    // TODO: rm when new version is in production
+    if (this.config.version === '1.0') {
+      delete data.collectionType;
```

Comment:
`collectionType` should neither be added to new set API version update requests.

Reply:
From the perspective of this Set API client, if the property is supplied in an update call, then it should be sent to the API.

We are not however supplying it in the update call arguments from the SetFormModal, so it won't be.